From: Markus Stockhausen <markus.stockhausen@gmx.de>
Subject: [PATCH] realtek: disable upstream Realtek board

Upstream now has integrated the Realtek target into the generic
MIPS initialization. For now disable the board compilation.

Submitted-by: Markus Stockhausen <markus.stockhausen@gmx.de>
---
--- a/arch/mips/generic/Makefile
+++ b/arch/mips/generic/Makefile
@@ -13,4 +13,4 @@ obj-$(CONFIG_LEGACY_BOARD_SEAD3)	+= boar
 obj-$(CONFIG_LEGACY_BOARD_OCELOT)	+= board-ocelot.o
 obj-$(CONFIG_MACH_INGENIC)			+= board-ingenic.o
 obj-$(CONFIG_VIRT_BOARD_RANCHU)		+= board-ranchu.o
-obj-$(CONFIG_MACH_REALTEK_RTL)		+= board-realtek.o
+# obj-$(CONFIG_MACH_REALTEK_RTL)		+= board-realtek.o
